CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 - Shiny link is now a toggle button in the top right of each Pokedex entry
 - Hide shiny link for Scarlet/Violet Pokemon since we don't have shiny images for them
 - Improve accessibility for Pokemon links so it's clearer
+- Use PokeAPI for Scarlet/Violet Pokemon ([@kittenchilly](https://github.com/kittenchilly))
```

Comment:
You're more than welcome to have the shout-out in the CHANGELOG, but I meant before to clarify: do you want to be in the "Special Thanks" section of the website? This is fantastic work.